v63002/inref/gtm8165.m

```diff
@@ -14,14 +14,14 @@
 writeslashwait
 	tstart ():(serial:transaction="BA")
 	if $trestart>2  do
-	. write:($job=$$^%PEEKBYNAME("node_local.in_crit","DEFAULT")) "DB Crit of DEFAULT owned by pid = ",$$^%PEEKBYNAME("node_local.in_crit","DEFAULT"),!
+	. write:($job=$$^%PEEKBYNAME("node_local.in_crit","DEFAULT")) "DB Crit of DEFAULT owned by pid = xxxx as expected",!
 	. set sock="gtm8165.socket"
 	. open sock:(LISTEN="gtm8165sock.socket:LOCAL":attach="handle")::"SOCKET"
 	. use sock
 	. if $ZTRNLNM("timeout")  write /wait(.999)
 	. if '$ZTRNLNM("timeout")  write /wait
 	. close sock
-	if $trestart<=2  do
+	if $trestart<=2  do ;cannot use else because this commit changes the value of $test, similar usages throughout the test
```

Comment:
We cannot use "else" because the write /wait with timeout done above sets $test, not because of the TCOMMIT done a little later (that does not set $test).